plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2952,8 +3050,8 @@ public void testSplitPruningFromDataFileStatistics(DataMappingTestSetup testSetu
             String tableName = table.getName();
             String values =
                     Stream.concat(
-                                    nCopies(100, testSetup.getSampleValueLiteral()).stream(),
-                                    nCopies(100, testSetup.getHighValueLiteral()).stream())
```

Comment:
My IDE (IntelliJ IDEA 2021.2.3 (Ultimate Edition)) would re-add this indentation back

Reply:
strange, mine did it on its own, but it should not be here, sorry

Reply:
no problem. ok to just revert, but we need to somehow make sure our IDEs work the same.